website/content/terminal/usage/basics/index.md

```diff
@@ -1,114 +1,280 @@
 ---
 title: Basics
-description: The OpenBB Terminal is a modern investment research platform for everyone. It is a desktop application that allows you to access all the data and tools you need to make better investment decisions.
-keywords: [basics, installation, commands, menus, your own data, introduction, openbb terminal, explanation, basic usage, usage, how to]
+description: An introduction to the The OpenBB Terminal, a Command Line Interface (CLI) application.  Functions (commands) are called through the keyboard with results returning as charts, tables, or text.
+keywords: [basics, commands, functions, features, menus, introduction, openbb terminal, obb, usage, how to, charts, tables, themes, styles, functions, data, sources, getting started]
 ---
+## Overview
 
-The OpenBB Terminal is based off the <a href="https://en.wikipedia.org/wiki/Command-line_interface" target="_blank" rel="noreferrer noopener">Command Line Interface (CLI)</a>
-which is installed by default on every computer. By opening the application you have installed from the [Installation Page](/terminal/installation/windows),
-you are greeted with the following interface:
+The OpenBB Terminal is a Command Line Interface (CLI) application.  Functions (commands) are called through the keyboard with results returned as charts, tables, or text.  Charts and tables (if enabled) are displayed in a new window, and are fully interactive, while text prints directly to the Terminal screen.  Commands are grouped into menus, with a menu or sub-menu being visually distinguishable from a function by the, `>`, on the far left of the screen.  The color of the text can be altered under the [`/settings` menu](https://docs.openbb.co/terminal/usage/guides/customizing-the-terminal).
 
-<img width="800" alt="image" src="https://user-images.githubusercontent.com/46355364/223211150-5c0bce44-98f3-403c-9db1-2344c1ad79d4.png"></img>
+Navigating through the Terminal menus is similar to climbing a tree, or traversing folders from any operating system's command line prompt.  The `/home` screen is the trunk and the menus are branches.  Instead of `C:\Users\OpenBB\Documents`, it is, [`/stocks/options`](https://docs.openbb.co/terminal/usage/intros/stocks/options). Instead of, `cd ..`, two periods - `..` - returns to the menu one level back towards the home screen.  Absolute paths are also valid to-and-from any point.  From the [`/stocks/options`](https://docs.openbb.co/terminal/usage/intros/stocks/options) menu, go directly to [`/crypto`](https://docs.openbb.co/terminal/usage/intros/crypto).  By itself, `/`, returns to the home level.
```

Comment:
"climbing a tree" not quite making sense

Reply:
How about, "walking down a path"?  The metaphor was a suggestion from Jerry to provide non-technical users with a way to visualize the directory tree in their minds.

Reply:
Hmmm.  ChatGPT suggests something like (dont like this either):

Think of the file tree as a system of folders and files organized in a hierarchical manner, where the folders are like drawers or cabinets, and the files are like documents or items stored inside those drawers or cabinets.

I like the use of a metaphor, but climbing a tree in my head doesnt exactly match to navigating a file tree

Reply:
Better?

```
Navigating through the Terminal menus is similar to following down a path, or traversing folders from any operating system's command line prompt.  The `/home` screen is the main path where everything begins, and the menus are paths branched from the main.
```

Reply:
Im not sure yet.

Reply:
So i told chatGPT to not use a tree or cabinet - and it used a book - So Terminal is a book - and each chapter is a menu.. etc

Reply:
A cabinet, who said anything about that?  I'd rather we make our own decisions, and not do what Lord ChatGPT says we should write.